### PR TITLE
Add GoBDokumente v1.6

### PR DESCRIPTION
**Pull request by mirkolenz**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/gobdokumente.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'gobdokumente' do
+  version '1.6'
+  sha256 'c271b090affa0d002c96704c1303253210d534230ed256521b352d4b7119ffe2'
+
+  # download.moapp.software was verified as official when first introduced to the cask
+  url 'https://download.moapp.software/GoBDokumente.zip'
```
Comment (vitorgalvao) [inline Casks/gobdokumente.rb]: Please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md): `https://sparkle.moapp.software/gobdokumente.xml`.

And in the future, please tick (or don’t untick) the box that allows maintainers to do edits directly. It makes life easier for everyone as we don’t have to waste valuable time with back and forth to fix things.

File: Casks/gobdokumente.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name 'GoBDokumente'
+  homepage 'https://gobdokumente.de/'
+
+  app 'GoBDokumente.app'
+
+  caveats 'The app is called "GoBDocuments" on English systems.'
```
Comment (vitorgalvao) [inline Casks/gobdokumente.rb]: Remove the caveat.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by MirkoLenz** [comment]
Hi! Sorry for all inconveniences caused. I thought I left the checkbox untouched to allow moderators to edit the pull request. I will make sure that this option is correctly set for my next PR.

As there was an issue with my fork of this repository (errors due to it being a shallow copy), I deleted it and did a new fork of it. Sadly, I do not know how to edit this PR anymore as my original fork is gone. Should I close this PR and create a new one with the updated formula?

Again, I am sorry for any inconveniences caused and really appreciate the time you all put into this project!

**Comment by vitorgalvao** [comment]
> Should I close this PR and create a new one with the updated formula?

That works. I’ll go ahead and close this one. Submit the new one whenever you’re ready.

> I am sorry for any inconveniences caused

Don’t fret about it. As long as we’re now on the same page, that’s what matters.

> and really appreciate the time you all put into this project!

Thank you for the kind words.

**Pull request closed without merging**